app/Models/EntryDAO.php

```diff
@@ -670,7 +670,8 @@ protected function sqlConcat($s1, $s2) {
 		return 'CONCAT(' . $s1 . ',' . $s2 . ')';	//MySQL
 	}
 
-	protected function sqlListEntriesWhere($alias = '', $filters = null, $state = FreshRSS_Entry::STATE_ALL, $order = 'DESC', $firstId = '', $date_min = 0) {
+	protected function sqlListEntriesWhere($alias = '', $filters = null, $state = FreshRSS_Entry::STATE_ALL,
+			$order = 'DESC', $firstId = '', $date_min = 0) {
```

Comment:
In case like that, don't you think that having one argument per line is more readable?
```php
protected function sqlListEntriesWhere(
	$alias = '',
	$filters = null,
	$state = FreshRSS_Entry::STATE_ALL,
	$order = 'DESC',
	$firstId = '',
	$date_min = 0
) {
```

Reply:
Potentially yes for here, although I am not so found of excessive lines in general, as it generates more vertical scrolling, which reduces my ability to get the bigger picture of a piece of code.
In any case, the ambition of this PR (which is already large :-)) was *just* to fix the biggest problems - there is still room for improvement